10/umbraco-cms/tutorials/multilanguage-setup.md

```diff
@@ -226,3 +226,105 @@ For viewing purposes, I've added a stylesheet to my website. The final result sh
 **German Version:**
 
 <figure><img src="images/final-result-da.png" alt=""><figcaption></figcaption></figure>
+
+
+## Using Muli languages across APIs
+
+When requesting content over an API, the culture will fallback to the default, unless explicitly set. 
+
+To do this, we can use the IVariationContextAccessor. 
+
+```csharp
+public class ExampleController : SurfaceController
+{
+    private readonly ILocalizationService _localizationService;
+    private readonly IVariationContextAccessor _variationContextAccessor;
+
+    public ExampleController(IUmbracoContextAccessor umbracoContextAccessor, IUmbracoDatabaseFactory databaseFactory, ServiceContext services, AppCaches appCaches, IProfilingLogger profilingLogger, IPublishedUrlProvider publishedUrlProvider, ILocalizationService localizationService, IVariationContextAccessor variationContextAccessor) : base(umbracoContextAccessor, databaseFactory, services, appCaches, profilingLogger, publishedUrlProvider)
+    {
+        _localizationService = localizationService;
+        _variationContextAccessor = variationContextAccessor;
+    }
+
+    public IActionResult Index(string culture = null)
+    {
```

Comment:
Hi @jonathoncove2 

I found the LanguageServiceTests.cs in the v14/dev repository and it does a Async call to get all the languages:
https://github.com/umbraco/Umbraco-CMS/blob/e4dbf4ce9ff9ca70ed7c82c0c735a4d576aad692/tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/LanguageServiceTests.cs#L15

Could you try and update this controller to inject the ILanguageService here and then use the LanguageService.GetAllAsync() function and see if it provides the same result as the localizationService?